ui/app/app.js

```diff
@@ -99,6 +99,9 @@ export default class App extends Application {
     sync: {
       dependencies: {
         services: ['flash-messages', 'router', 'store', 'version'],
+        externalRoutes: {
+          secret: 'vault.cluster.secrets.backend.kv.secret.details',
```

Comment:
I mentioned this below - but I think we should make this name more specific to kv v2 so we don't have to rename if/when we accommodate other secret engine types